scripts/encapp_tool/adb_cmds.py

```diff
@@ -55,13 +55,16 @@ def get_device_info(serial_inp: Optional[str], debug=0) -> Tuple[Dict, str]:
     if serial_inp is None:
         # if user did not select a serial_inp, make sure there is only one
         # device available
-        assert len(device_info) == 1, f"error: need to choose a device [{', '.join(device_info.keys())}]"
+        assert (
```

Comment:
suggest to move all lint changes to a separate patch (it makes it easier for reviewers)